example-templates/buy/add.html

```diff
@@ -30,12 +30,10 @@
     {% else %}
         {% set product = craft.products.one() %}
         <div class="m-10">
-        <a href="/buy/add?purchasableId={{product.defaultVariant.id}}"class="bg-blue-commerce text-white w-1/2 rounded px-3 py-3 ml-1 mt-6 text-sm hover:bg-blue-dark outline-0">
+        <a href="{{ url('buy/add?purchasableId=' ~ product.defaultVariant.id) }}" class="bg-blue-commerce text-white w-1/2 rounded px-3 py-3 ml-1 mt-6 text-sm hover:bg-blue-dark outline-0">
```

Comment:
Let's move to using the param features of the url function:

```twig
{{ url('buy/add', {'purchasableId': product.defaultVariant.id}) }}
```

Reply:
Hey @pdaleramirez  Can you also make sure to update the `example-templates/shop` templates also. I only see updates to the `example-templates/buy` folder of templates.

Thanks :)

Reply:
@lukeholder updated it on latest commit.